pages/api/places/index.js

```diff
@@ -1,16 +1,27 @@
 // import { places } from "../../../lib/db";
-import Places from "@/db/models/Places";
-import dbConnect from "@/db/models/Connect.js";
+import Place from "@/db/models/Places";
+import dbConnect from "@/db/models/connect.js";
 
 export default async function handler(request, response) {
   await dbConnect();
 
   if (request.method === "GET") {
     try {
-      const places = await Places.find();
+      const places = await Place.find();
       return response.status(200).json(places);
     } catch (error) {
       return response.status(500).json({ message: "Error fetching places" });
     }
   }
+
+  if (request.method === "POST") {
+    console.log("incoming", request.body);
```

Comment:
you could think of removing the console.log

Reply:
done